docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+received. Usually, it is made by the FIWARE Scrum Master together with
+the process to assign the ticket to the corresponding owner in order to
+be resolved.
+
+![HD-Chapter, HD-Enabler and HD-Node attributes in issues.](image22.png)
+
+For the assignee of the JIRA ticket, we have to differentiate between
+Generic Enablers owners and FIWARE Lab administrators’ nodes. For the
+first one, the list of owners can be obtained from the following table:
+
+[*https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit\#gid=694707434*](https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit#gid=694707434)
+
+It is a working document of FIWARE and it is available for the FIWARE
+Lab administrators. If a FIWARE Lab node administrators want to get
+access to it, they can request access to the owner of the file (fernando
+dot lopez at fiware dot org). In case of the FIWARE Lab administrator
+owners this information should be found in the following table:
+
+[*https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit\#gid=744561338*](https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit#gid=744561338)
+
+Regarding the components, usually they are automatically selected by the
+different tools that are behind JIRA, but in case that we found a JIRA
+ticket without the component the available values are the following:
+
+
+
+| **Components** | **Description** | 
+| -- | -- | 
+| FIWARE-COLLABORATION-REQ | Issues related to the request associated to some type of collaboration with FIWARE in terms of participation in some events or in terms of improvement some FIWARE GEri |
+| FIWARE-FEEDBACK | General issues related to feedback recover from the users. |
+| FIWARE-GENERAL-HELP | Issues in general not classified in the other components. |
+| FIWARE-LAB-HELP | Issues related to some of the FIWARE Lab nodes |
+| FIWARE-MUNDUS-REQ | Issues regarding the activities of FIWARE associated to the extension of FIWARE beyond Europe (FI-GLOBAL project). |
+| FIWARE-OPEN-DATA-REQ | (Deprecated) Issues related to the management of Open Data inside CKAN tool or related to the possibility to contribute with Open Data inside FIWARE ecosystem. |
+| FIWARE-OPS-HELP | Issues related to the use of the different FIWARE Ops tools.
+| FIWARE-SMART-CITIES-REQ | (Deprecated) Issues related to the collaboration in the SmartCities |
+| FIWARE-SPEAKERS-REQ | Issues related to the request to get some FIWARE expert to provide some presentation or speech in some events or just in some Summer School or so on. |
+| FIWARE-TECH-HELP | Issues related to some of the FIWARE GEri. |
+| FIWARE-TRAINING-REQ | (Deprecated) Issues related to the request of training for some of the accelerator program. Currently, it is not needed due to there is no request for training from accelerator programs. |
+
+In case of HD-Chapter and HD-Enabler, they make reference to the
```

Comment:
In case of HD-Enabler, it makes reference to the
corresponding Enabler to which have to be assigned the
corresponding JIRA ticket, how it was described previously. It is
something that have to be selected once we edit the corresponding JIRA
issue.

Reply:
FIWARE GEri -> FIWARE GE